examples/src/app/helpers/example-data.mjs

```diff
@@ -1,38 +1,48 @@
-import exampleData from '../../../dist/example-data.js';
-
-function capitalizeFirstLetter(string) {
-    return string.charAt(0).toUpperCase() + string.slice(1);
-}
-
+import { exampleData } from '../../example-data.mjs';
+import { kebabCaseToPascalCase } from './strings.mjs';
+/** @type {Record<string, Record<string, object>>} */
 const categories = {};
+/** @type {Record<string, object>} */
 const paths = {};
-
 Object.keys(exampleData).forEach((categorySlug) => {
-    const category = categorySlug.split('-').map(a => capitalizeFirstLetter(a)).join('');
+    const category = kebabCaseToPascalCase(categorySlug);
     categories[categorySlug] = {
         examples: {}
     };
     Object.keys(exampleData[categorySlug]).forEach((exampleSlug, i) => {
-        const name = exampleSlug.split('-').map(a => capitalizeFirstLetter(a)).join('').replace('1d', '1D').replace('2d', '2D');
+        const name = kebabCaseToPascalCase(exampleSlug);
+        /*
+        let realClass;
+        try {
+            realClass = realExamples[category][name + "Example"];
+        } catch (e) {
+            // slow process of porting everything to proper MJS
+            debugger;
+            return;
+        }
+        if (!realClass) {
+            return;
+        }
+        */
```

Comment:
remove this unless useful

Reply:
Gonna remove it, only comes in handy a bit later for 100% build step free ES6 version